api/src/pcapi/routes/backoffice_v3/accounts.py

```diff
@@ -625,8 +630,9 @@ def _get_steps_tunnel_age18(
                 if id_check_history.type
                 in {
                     fraud_models.FraudCheckType.UBBLE.value,
-                    fraud_models.FraudCheckType.DMS.value,
+                    fraud_models.FraudCheckType.EDUCONNECT.value,
                     fraud_models.FraudCheckType.JOUVE.value,
+                    fraud_models.FraudCheckType.DMS.value,
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Tu n'ajoutes pas `and users_models.EligibilityType.AGE18.value in id_check_history.applicable_eligibilities` comme pour les autres `_get_steps_***`?

Reply:
Effectivement je l'ai oublié !